tests/pds/pds_storagesize_increase_test.go

```diff
@@ -0,0 +1,916 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	pds "github.com/portworx/pds-api-go-client/pds/v1alpha1"
+	pdsdriver "github.com/portworx/torpedo/drivers/pds"
+	"github.com/portworx/torpedo/drivers/pds/controlplane"
+	pdslib "github.com/portworx/torpedo/drivers/pds/lib"
+	pdsbkp "github.com/portworx/torpedo/drivers/pds/pdsbackup"
+	restoreBkp "github.com/portworx/torpedo/drivers/pds/pdsrestore"
+	tc "github.com/portworx/torpedo/drivers/pds/targetcluster"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	v1 "k8s.io/api/apps/v1"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+var _ = Describe("{ResizeStorageAndPerformRestoreXFSRepl2}", func() {
```

Comment:
@jkagliwal-px would it be possible to use the repl factor and filesystem as config inputs if there are no other flow changes, that way we can have one method and drive the same using different inputs.

Reply:
@atrivedi-px  The implementation is holding same concept of using a single struct with multiple inputs. However at this point we cannot add it in the params-input / Config file, as it will lead to framework changes. 
We can discuss more on this, if needed.

Reply:
Done!